installation/downloadReferences.sh

```diff
@@ -1,66 +1,105 @@
+#!/bin/bash
 
+set -o pipefail
+
+# which files to download
 REFERENCE_GENOME=true
 dbSNP_FILE=true
 MAPPABILITY_FILE=true
 CHROMOSOME_LENGTH_FILE=true
 statFiles=true
-IMPUTE_FILES=false
-
-SCRIPT=$(readlink -f $0)
-scriptdir=`dirname $SCRIPT`
+IMPUTE_FILES=true
 
-echo $scriptdir
+exit_on_fail() {
```

Comment:
What about setting `set -e` and installing a trap on ERR

```bash
trap { echo "Download incomplete. ..."; } ERR
```
?

You would not need the calls to exit_on_fail and the `&&` chains, which are easy to forget anyway.